XML/TestCases/BenchmarkTests-MPI.xml

```diff
@@ -0,0 +1,47 @@
+<TestCases>
+<!-- MVAPICH MPI -->
+    <test>
+		<testName>INFINIBAND-MVAPICH-MPI-OSU-2VM</testName>
+		<testScript>VERIFY-INFINIBAND-MultiVM.ps1</testScript>
+		<setupType>TwoVM1Dep</setupType>
+		<OverrideVMSize>Standard_HC44rs</OverrideVMSize>
+		<files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\TestRDMA_MultiVM.sh,.\Testscripts\Linux\SetupRDMA.sh</files>
+		<Priority>3</Priority>
+		<TestParameters>
+			<param>install_ofed="INFINIBAND_INSTALL_OFED"</param>
+			<param>install_ofed_from_extension="INFINIBAND_INSTALL_OFED_FROM_EXTENSION"</param>
+			<param>num_reboot="INFINIBAND_MVAPICH_TOTAL_REBOOT_COUNT"</param>
+			<param>mpi_settings="INFINIBAND_MVAPICH_MPI_SETTINGS_IB"</param>
+			<param>ib_nic="INFINIBAND_IB_NIC"</param>
+			<param>imb_mpi1_tests="INFINIBAND_MVAPICH_MPI_IMB-TESTNAMES"</param>
+			<param>mpi1_ppn=INFINIBAND_MVAPICH_PPN</param>
+			<param>imb_mpi1_tests_iterations=INFINIBAND_MPI_TEST_COUNT</param>
+			<param>imb_rma_tests="INFINIBAND_MVAPICH_RMA_TESTNAMES"</param>
+			<param>rma_ppn=INFINIBAND_MVAPICH_PPN</param>
+			<param>imb_rma_tests_iterations=INFINIBAND_MVAPICH_RMA_TEST_COUNT</param>
+			<param>imb_nbc_tests="INFINIBAND_MVAPICH_NBC_TESTNAMES"</param>
+			<param>nbc_ppn=INFINIBAND_MVAPICH_PPN</param>
+			<param>imb_nbc_tests_iterations=INFINIBAND_MVAPICH_NBC_TEST_COUNT</param>
+			<param>p2p_ppn=INFINIBAND_MVAPICH_PPN</param>
+			<param>imb_p2p_tests_iterations=INFINIBAND_MVAPICH_P2P_TEST_COUNT</param>
+			<param>io_ppn=INFINIBAND_MVAPICH_PPN</param>
+			<param>imb_io_tests_iterations=INFINIBAND_MVAPICH_IO_TEST_COUNT</param>
+            <param>osu_p2p_tests=INFINIBAND_MVAPICH_OSU_P2P_TESTNAMES</param>
+            <param>osu_p2p_tests_iterations=INFINIBAND_MVAPICH_OSU_P2P_TEST_COUNT</param>
+            <param>mpi_type="mvapich"</param>
+			<param>mlx_ofed_partial_link=MLX_OFED_PARTIAL_LINK</param>
+			<param>ibm_platform_mpi=IBM_PLATFORM_MPI_DOWNLOAD</param>
+			<param>intel_mpi=INTEL_MPI_DOWNLOAD</param>
+			<param>open_mpi=OPEN_MPI_DOWNLOAD</param>
+			<param>mvapich_mpi=MVAPICH_MPI_DOWNLOAD</param>
+			<param>intel_mpi_benchmark=INTEL_MPI_BENCHMARK_DOWNLOAD</param>
+            <param>osu_mpi_benchmark=OSU_MPI_BENCHMARK_DOWNLOAD</param>
```

Comment:
We have MVAPICH MPI in the setup/test. Is 'OSU MVAPICH' different from the existing MVAPICH MPI? Otherwise, it looks osu_mpi_benchmark is duplicated to mvapich_mpi. Please clarify. I will review the rest after I get your answer.

Reply:
OSU benchmarks are included as part of mvapich installation. However for other mpi types (intel mpi, open mpi etc.) the osu mpi benchmark needs to be installed separately. To keep the tests consistent for all mpi types I am downloading benchmarks here. An alternative would be to conditionally use the preinstalled benchmarks in case of mvapich. I can do the change.

Reply:
Added a condition to install only in non-mvapich mpi types.